bin/run-migration-job.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+This script will either migrate or rollback the swoop database to a specified
+migration version. The migration version, action type (of either migrate/rollback),
+and an override parameter are all specified in the values.yml from the Swoop-DB helm
+chart.
+
+It requires the following environment variables be set:
+
+    - PGHOST:
+          Postgres K8s service name which will be used to connect to the database
+    - PGUSER:
+          name of the user (role) that will perform the migrations
+    - PGPORT:
+          port number of the database container
+    - PGDATABASE: name of the database which will be used for migrations
+    - Any additional libpq-supported connection parameters
+          (https://www.postgresql.org/docs/current/libpq-envars.html)
+"""
+import argparse
+import asyncio
+import os
+import sys
+
+from buildpg import V, funcs, render
+
+from swoop.db import SwoopDB
+
+APPLICATION_ROLES = ["user_api", "user_caboose", "user_conductor"]
+
+
+def stderr(*args, **kwargs) -> None:
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+def check_positive(value) -> int:
+    ivalue = int(value)
+    if ivalue <= 0:
+        raise argparse.ArgumentTypeError(
+            "%s is an invalid migration version number. Only \
+                positive values are supported."
+            % value
+        )
+    return ivalue
```

Comment:
Also you don't really _need_ this check. Non-positive values won't ever resolve to a valid migration path, so I would expect you can simply let the migration function return an error when the user provides a silly value. Not that you need to remove this check.

But again, you string has a bunch of whitespace in it. Also I would discourage the old-school `%` formatting. If you really want a multiline string:

```suggestion
    if ivalue <= 0:
        raise argparse.ArgumentTypeError(
            f"{value} is an invalid migration version number. "
            "Only positive values are supported.",
        )
    return ivalue
```